Controls/Grid/Grid.UWP/View/Services/Editing/EditingService.cs

```diff
@@ -118,18 +118,20 @@ public bool CommitEdit(ActionTrigger trigger)
                         if (trigger != ActionTrigger.ExternalEditor)
                         {
                             this.Owner.ExternalEditor.CommitEdit();
+                            this.Owner.CurrencyService.RefreshCurrentItem(true);
```

Comment:
Why is this needed? Did you find any issue with current item scrolling out of view?

Reply:
See https://github.com/telerik/UI-For-UWP/pull/513#issuecomment-1283460780